### DISCUSSION TITLE
Make some Alert/Beep when Errors too many

### DISCUSSION THREAD

**Discussion by dengyakui** [datalust/seq-tickets#1538] [General]
We Have a big monitor screen to show Seq’s real-time log, but nobody will always see the screen when coding
In order to get immediate notification and move to see the monitor,
can Seq make some BIG Alert Music ❤️‍🔥  when there were> 100 errors in a short time.
like:  
- 100 new [efcore] errors/ s
- 1000 new [some SourceContext or event type] errors/hour
 
I know email is a good way, but everybody doesn’t want to open email when at working 😂
Actually, today we had met some DeadLock Problems in SQLServer, and efcore has logged so many errors, but no one get notified before some customers contacted us.
![image](https://user-images.githubusercontent.com/3945919/161194980-21b1f9e1-e5bc-4158-80f5-a969c0829969.png)

**Comment by liammclennan** [comment]
Hi @dengyakui ,

Seq already has the ability to trigger an alert based on a condition like the example you provided. 

If you have an audio device with an API you could connect it with the Seq.App.HttpRequest app.

**Reply by dengyakui** [reply]
Hi,@liammcleanman:
Thanks for the response. After check the link [https://github.com/datalust/seq-app-httprequest](url), I think i can set up.

**Comment by liammclennan** [comment]
You're welcome. Let me know how it goes.
